### DISCUSSION TITLE
System.Text.Json dependency version

### DISCUSSION THREAD

**Discussion by IldarKhayrutdinov** [Mapsui/Mapsui#2808] [General]
`<PackageVersion Include="System.Text.Json" Version="[8.0.5,9.0.0)" />`
Is is possible upgrade to [9.0.0](https://www.nuget.org/packages/System.Text.Json/9.0.0#supportedframeworks-body-tab)?
Supports .net8.0 and .net9.0.

**Comment by inforithmics** [comment]
Yes, it is possible to upgrade it to 9.0.0 since 9.0.0 is now released

**Reply by inforithmics** [reply]
pull request, to see if everything runs through.
https://github.com/Mapsui/Mapsui/pull/2809
